ghtt/assignment.py

```diff
@@ -172,16 +174,16 @@ def create_pr(ctx, branch, title, body, source, yes, students=None, groups=None,
             continue
 
         if not branch_already_pushed:
-            command = ["git", "push", g_repo.ssh_url, "master:{}".format(branch)]
+            command = ["git", "push", g_repo.ssh_url, f"{g_repo.master_branch or default_branch}:{branch}"]
```

Comment:
Always create PR to specified default branch; to make `ghtt` behavior more predictable regardless or remote state.

```suggestion
            command = ["git", "push", g_repo.ssh_url, f"{default_branch}:{branch}"]
```